components/dashboard/src/projects/NewProject.tsx

```diff
@@ -83,7 +83,7 @@ export default function NewProject() {
                 <div className="mt-12 flex space-x-2">
                     <a href={`/projects/${Project.slug(project)}/settings`}>
                         <Button type="secondary" onClick={onNewWorkspace}>
-                            Enable Prebuilds
+                            Go to Project Settings
```

Comment:
To be consistent with the style of the other button (<kbd>Create Workspace</kbd>), I propose we simplify to "Project Settings".
```suggestion
                            Project Settings
```

Reply:
Thanks, @filiptronicek. Opted for using the prefix to avoid any confusion as this is a button used erroneously for navigation, and follow the pattern we already use in workspace start page, etc (see _Go to dashboard_, etc.) Let's update these all in one step once we come up with a better pattern. From https://github.com/gitpod-io/gitpod/pull/18007#pullrequestreview-1510776080:

> **Links are for navigation. Buttons are for everything else.**
> We should avoid mixing these when possible.

> Alternatively, having a button component variant that can visually style an action like an anchor link when using a plain link isn't possible, sounds great. ✔️

Reply:
Thanks for the explanation!